src/SWP/Bundle/MultiTenancyBundle/Doctrine/ORM/OrganizationRepository.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * @author Rafał Muszyński <rafal.muszynski@sourcefabric.org>
+ * @copyright 2015 Sourcefabric z.ú.
+ * @license http://www.gnu.org/licenses/gpl-3.0.txt
+ */
+namespace SWP\Bundle\MultiTenancyBundle\Doctrine\ORM;
+
+use SWP\Bundle\StorageBundle\Doctrine\ORM\EntityRepository;
+use SWP\Component\MultiTenancy\Repository\OrganizationRepositoryInterface;
+
+class OrganizationRepository extends EntityRepository implements OrganizationRepositoryInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function findByName($name)
```

Comment:
I would rename these methods to findEnabledByName, etc. Unless you always want to get only enabled entities, in which case, you could use a filter.